app/stories/pix-input.stories.mdx

```diff
@@ -10,7 +10,7 @@ Le PixInput permet de créer un champ de texte.
 
 ## Accessibilité
 
-Si vous utilisez le `PixInput` sans label alors il faut renseigner le paramètre `ariaLabel`, sinon le composant renvoie une erreur.
+Si vous utilisez le `PixInput` sans label alors il faut renseigner le paramètre `ariaLabel`. Il est possible d'ajouter un label en dehors du composant en précisant bien un attribut `for` correspondant à l'`@id` passé au PixInput.
```

Comment:
Pour info, est-ce que ce qu'on pourrait aussi le faire avec un `aria-labelledby` ?

Reply:
Le `aria-labelledby` est similaire à ajouter un `label` avec un `for` :)